--- .engine/build/mach_initialize.py	2023-07-07 08:15:29.000000000 +1000
+++ src/build/mach_initialize.py	2023-07-09 16:45:59.220019501 +1000
@@ -155,21 +155,6 @@
 
     # Centralized registry of available mach commands
     MACH_COMMANDS = {
-        "valgrind-test": MachCommandReference("build/valgrind/mach_commands.py"),
-        "devtools-css-db": MachCommandReference(
-            "devtools/shared/css/generated/mach_commands.py"
-        ),
-        "webidl-example": MachCommandReference("dom/bindings/mach_commands.py"),
-        "webidl-parser-test": MachCommandReference("dom/bindings/mach_commands.py"),
-        "hazards": MachCommandReference(
-            "js/src/devtools/rootAnalysis/mach_commands.py"
-        ),
-        "reftest": MachCommandReference("layout/tools/reftest/mach_commands.py"),
-        "jstestbrowser": MachCommandReference("layout/tools/reftest/mach_commands.py"),
-        "crashtest": MachCommandReference("layout/tools/reftest/mach_commands.py"),
-        "android": MachCommandReference("mobile/android/mach_commands.py"),
-        "gradle": MachCommandReference("mobile/android/mach_commands.py"),
-        "gradle-install": MachCommandReference("mobile/android/mach_commands.py"),
         "mach-commands": MachCommandReference(
             "python/mach/mach/commands/commandinfo.py"
         ),
@@ -256,113 +241,6 @@
         "release": MachCommandReference(
             "python/mozrelease/mozrelease/mach_commands.py"
         ),
-        "remote": MachCommandReference("remote/mach_commands.py"),
-        "puppeteer-test": MachCommandReference("remote/mach_commands.py"),
-        "generate-test-certs": MachCommandReference(
-            "security/manager/tools/mach_commands.py"
-        ),
-        "taskgraph": MachCommandReference("taskcluster/mach_commands.py"),
-        "taskcluster-load-image": MachCommandReference("taskcluster/mach_commands.py"),
-        "taskcluster-build-image": MachCommandReference("taskcluster/mach_commands.py"),
-        "taskcluster-image-digest": MachCommandReference(
-            "taskcluster/mach_commands.py"
-        ),
-        "release-history": MachCommandReference("taskcluster/mach_commands.py"),
-        "awsy-test": MachCommandReference("testing/awsy/mach_commands.py"),
-        "fetch-condprofile": MachCommandReference(
-            "testing/condprofile/mach_commands.py"
-        ),
-        "run-condprofile": MachCommandReference("testing/condprofile/mach_commands.py"),
-        "firefox-ui-functional": MachCommandReference(
-            "testing/firefox-ui/mach_commands.py"
-        ),
-        "geckodriver": MachCommandReference("testing/geckodriver/mach_commands.py"),
-        "addtest": MachCommandReference("testing/mach_commands.py"),
-        "test": MachCommandReference("testing/mach_commands.py"),
-        "cppunittest": MachCommandReference("testing/mach_commands.py"),
-        "jstests": MachCommandReference("testing/mach_commands.py"),
-        "jit-test": MachCommandReference("testing/mach_commands.py"),
-        "jsapi-tests": MachCommandReference("testing/mach_commands.py"),
-        "jsshell-bench": MachCommandReference("testing/mach_commands.py"),
-        "cramtest": MachCommandReference("testing/mach_commands.py"),
-        "test-info": MachCommandReference("testing/mach_commands.py"),
-        "rusttests": MachCommandReference("testing/mach_commands.py"),
-        "fluent-migration-test": MachCommandReference("testing/mach_commands.py"),
-        "marionette-test": MachCommandReference("testing/marionette/mach_commands.py"),
-        "mochitest": MachCommandReference("testing/mochitest/mach_commands.py"),
-        "geckoview-junit": MachCommandReference("testing/mochitest/mach_commands.py"),
-        "mozharness": MachCommandReference("testing/mozharness/mach_commands.py"),
-        "raptor": MachCommandReference("testing/raptor/mach_commands.py"),
-        "raptor-test": MachCommandReference("testing/raptor/mach_commands.py"),
-        "talos-test": MachCommandReference("testing/talos/mach_commands.py"),
-        "tps-build": MachCommandReference("testing/tps/mach_commands.py"),
-        "web-platform-tests": MachCommandReference(
-            "testing/web-platform/mach_commands.py"
-        ),
-        "wpt": MachCommandReference("testing/web-platform/mach_commands.py"),
-        "web-platform-tests-update": MachCommandReference(
-            "testing/web-platform/mach_commands.py"
-        ),
-        "wpt-update": MachCommandReference("testing/web-platform/mach_commands.py"),
-        "wpt-manifest-update": MachCommandReference(
-            "testing/web-platform/mach_commands.py"
-        ),
-        "wpt-serve": MachCommandReference("testing/web-platform/mach_commands.py"),
-        "wpt-metadata-summary": MachCommandReference(
-            "testing/web-platform/mach_commands.py"
-        ),
-        "wpt-metadata-merge": MachCommandReference(
-            "testing/web-platform/mach_commands.py"
-        ),
-        "wpt-unittest": MachCommandReference("testing/web-platform/mach_commands.py"),
-        "wpt-test-paths": MachCommandReference("testing/web-platform/mach_commands.py"),
-        "wpt-fission-regressions": MachCommandReference(
-            "testing/web-platform/mach_commands.py"
-        ),
-        "xpcshell-test": MachCommandReference("testing/xpcshell/mach_commands.py"),
-        "telemetry-tests-client": MachCommandReference(
-            "toolkit/components/telemetry/tests/marionette/mach_commands.py"
-        ),
-        "data-review": MachCommandReference(
-            "toolkit/components/glean/build_scripts/mach_commands.py"
-        ),
-        "perf-data-review": MachCommandReference(
-            "toolkit/components/glean/build_scripts/mach_commands.py"
-        ),
-        "update-glean-tags": MachCommandReference(
-            "toolkit/components/glean/build_scripts/mach_commands.py"
-        ),
-        "update-glean": MachCommandReference(
-            "toolkit/components/glean/build_scripts/mach_commands.py"
-        ),
-        "browsertime": MachCommandReference("tools/browsertime/mach_commands.py"),
-        "compare-locales": MachCommandReference(
-            "tools/compare-locales/mach_commands.py"
-        ),
-        "l10n-cross-channel": MachCommandReference("tools/lint/mach_commands.py"),
-        "busted": MachCommandReference("tools/mach_commands.py"),
-        "pastebin": MachCommandReference("tools/mach_commands.py"),
-        "mozregression": MachCommandReference("tools/mach_commands.py"),
-        "node": MachCommandReference("tools/mach_commands.py"),
-        "npm": MachCommandReference("tools/mach_commands.py"),
-        "logspam": MachCommandReference("tools/mach_commands.py"),
-        "doc": MachCommandReference("tools/moztreedocs/mach_commands.py"),
-        "install-moz-phab": MachCommandReference("tools/phabricator/mach_commands.py"),
-        "power": MachCommandReference("tools/power/mach_commands.py"),
-        "try": MachCommandReference("tools/tryselect/mach_commands.py"),
-        "import-pr": MachCommandReference("tools/vcs/mach_commands.py"),
-        "test-interventions": MachCommandReference(
-            "testing/webcompat/mach_commands.py"
-        ),
-        "esmify": MachCommandReference("tools/esmify/mach_commands.py"),
-        "xpcshell": MachCommandReference("js/xpconnect/mach_commands.py"),
-        "uniffi": MachCommandReference(
-            "toolkit/components/uniffi-bindgen-gecko-js/mach_commands.py"
-        ),
-        "storybook": MachCommandReference(
-            "browser/components/storybook/mach_commands.py"
-        ),
-        "widgets": MachCommandReference("toolkit/content/widgets/mach_commands.py"),
     }
 
     # Set a reasonable limit to the number of open files.
